src/main/java/com/minecolonies/colony/ColonyView.java

```diff
@@ -219,6 +220,14 @@ public int getMaxCitizens()
         return Collections.unmodifiableMap(citizens);
     }
 
+    public Collection<WorkOrderView> getWorkOrders()
+    {
+        return Collections.unmodifiableCollection(workOrders.values());
+    }
+
+    //todo send message to server for delete
+    //todo send message to server for change.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)